src/styles/modal.css

```diff
@@ -0,0 +1,108 @@
+/* modal */
+
+.ui.modal {
+  background-color: var(--shade-white);
+  border-radius: 0;
+  box-shadow: var(--box-shadow-5);
+}
+
+.ui.modal > .icon:first-child + * {
+  border-radius: 0;
+}
+
+/* close icon */
+
+.ui.modal > .close {
+  color: var(--shade-white);
+  font-size: 1.5rem;
+  height: 1.5rem;
+  padding: 0;
+  right: var(--spacing-2);
+  top: var(--spacing-3);
+  transition: opacity var(--transition-hover);
+  width: 1.5rem;
+}
+
+@media only screen and (max-width: 991px) {
+  .ui.modal > .close {
+    color: var(--shade-white);
+    font-size: 1.5rem;
+    height: 1.5rem;
+    padding: 0;
+    right: var(--spacing-2);
+    top: var(--spacing-3);
+    transition: opacity var(--transition-hover);
+    width: 1.5rem;
+  }
+}
+
+@media only screen and (max-width: 767px) {
+  .ui.modal > .close {
+    color: var(--shade-white);
+    font-size: 1.5rem;
+    height: 1.5rem;
+    padding: var(--spacing-2) 0;
+    right: var(--spacing-2);
+    top: var(--spacing-3);
+    transition: opacity var(--transition-hover);
+    width: 1.5rem;
+  }
+
+  .ui.modal > .actions {
+    margin-bottom: var(--spacing-3);
+  }
+
+  .ui.modal .content > .description {
+    padding: 0 !important;
+  }
+
+  .ui.modal > .content {
+    padding: var(--spacing-3) !important;
+  }
+}
+
+/* title */
+
+.ui.modal > .header {
+  background: var(--primary-pms-539-900);
+  border-bottom: none;
+  color: var(--shade-white);
+  padding: var(--spacing-2) var(--spacing-3);
+}
+
+.ui.modal > .header:not(.ui) {
+  font-family: var(--gotham);
+  font-size: var(--h2-font-size);
+  font-weight: 700;
+  line-height: var(--h2-line-height);
+}
+
+.ui.modal > .header > .ui.header {
+  color: var(--shade-white);
+  font-weight: 700;
+  margin: 0;
+}
+
+/* content */
+
+.ui.modal > .content {
+  background: var(--shade-white);
+  font-family: var(--gotham);
+  padding: var(--spacing-3);
+}
+
+/* actions */
+
+.ui.modal > .actions {
+  background: var(--shade-white);
+  border-top: 0;
+  display: flex;
+  padding: var(--spacing-3);
```

Comment:
I think we need this:

```
padding: 0 var(--spacing-3) var(--spacing-3) var(--spacing-3) !important;
```

Reply:
It might only be for small screens. I see it being applied on desktop 🤔